ui/components/navbar.js

```diff
@@ -18,10 +25,12 @@ function Navbar() {
               OMB Policy Library <sup>BETA</sup>
           </h1>
         </a>
-        <SearchContainer />
+        { searchContainer }
       </div>
     </div>
   );
 }
 
-export default Navbar;
+Navbar.propTypes = {
+  showSearch: PropTypes.bool
+};
```

Comment:
Should `showSearch` have a default of `true`?

Reply:
@cmc333333 Added the default value!